src/test/java/kgtestpack/KgHomeTest.java

```diff
@@ -96,7 +96,7 @@ public void checkout() {
 		boolean verifyFilter = men.verifyFilters();
 		Assert.assertTrue(verifyFilter);//assert the url after filtering
 		logger.info("Men's shoes are filtered and displayed based on the size and colour");
-		
+		logger.info("Method name is changed");
```

Comment:
This is not required here. Please remove it.

Reply:
I will correct it